pkg/operator/operator.go

```diff
@@ -1186,13 +1186,16 @@ func newRunReportForError(reason string, err error) runReport {
 		},
 	}
 }
+
+//nolint:unused
 func (r runReport) isDegraded() bool {
```

Comment:
This and `isUnavailable` are only used in `operator_test.go` which is why golint misses this I suppose. Should those functions live in the test file?

Reply:
> Should those functions live in the test file?

Yes, that is good idea! I would covert this to a utility function in test since it makes the test readable.